src/style.css

```diff
@@ -0,0 +1,103 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  font-style: normal;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.label::before {
+  position: absolute;
+  content: "";
+}
+
+.label-big::before {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.label-big:hover::before {
+  cursor: pointer;
+}
+
+.label-small::before {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.label-small:hover::before {
+  cursor: pointer;
+}
+
+.input {
+  width: 100%;
+  font-family: Avenir, sans-serif;
```

Comment:
Remove this line.

Reply:
or set `inherit` value to make it work and explicitly set the same value as for the entire page